client/dashboard/page-layout/style.scss

```diff
@@ -6,6 +6,11 @@
 	padding: $grid-unit-60 $grid-unit-20;
 }
 
+.dashboard-page-layout__title {
+	font-family: var(--dashboard-page-layout__title-font-family) !important;
```

Comment:
This feels hacky to me. I’d much rather have variables declared on the Core components themselves that we can then override from the dashboard-specific "theme file".

Reply:
I switched to be consistent with how we were targeting the description. No strong feelings about that though.

Reply:
I think using classnames should be the last resort because it means we're creating "custom components" but hidden in local files.